wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -82,6 +82,9 @@ class SwiftGenerator private constructor(
   private val encoder = DeclaredTypeName.typeName("Swift.Encoder")
   private val writableKeyPath = DeclaredTypeName.typeName("Swift.WritableKeyPath")
 
+  // Always fully-qualified in case a message type is named `Data`
```

Comment:
https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html

```suggestion
  // Always fully-qualified in case a message type is named `Data`.
```